src/Http/Controllers/Api/v1/CalendarApiController.php

```diff
@@ -0,0 +1,59 @@
+<?php
+namespace Seat\Kassie\Calendar\Http\Controllers\Api\v1;
+use Seat\Api\Http\Controllers\Api\v2\ApiController;
+use Seat\Kassie\Calendar\Models\Operation;
+/**
+ * Class CalendarApiController
+ * @package Seat\Kassie\Calendar\Http\Controllers\Api\v1
+ */
+class CalendarApiController extends ApiController
+{
+    /**
+     * @SWG\Get(
+     *     path="/calendar/operations",
+     *     tags={"Calendar"},
+     *     summary="Get a list of all calendar operations",
+     *     description="Returns list of all calendar operations",
+     *     security={"ApiKeyAuth"},
+     *     @SWG\Response(response=200, description="Successful operation",
+     *          @SWG\Schema(
+     *              type="array",
+     *              @SWG\Items(ref="#/definitions/Operation")
+     *          ),
+     *          examples={"application/json":
+     *              {
+     *                  {"title": "MOON MINING", "start_at": null, "end_at": null, "importance": "1", "description": "20% tax to Mining Lord", "staging_sys": "Jita", "staging_sys_id": 30000000, "staging_info": null, "is_cancelled": 0, "fc": "The Mittani", "fc_character_id": 12413371, "role_name": "Members" }
+     *              }
+     *          }
+     *     ),
+     *     @SWG\Response(response=400, description="Bad request"),
+     *     @SWG\Response(response=401, description="Unauthorized"),
+     *    )
+     *
+     * @return \Illuminate\Http\Resources\Json\AnonymousResourceCollection
+     */
+    public function getAllOperations()
+    {
+        $operations = Operation::all();

```

Comment:
Could you use the Laravel Resource and paginate result in order to return response ?
So the pagination is managed by the Framework based on request parameters :)

You got some sample in core like the endpoint used to return Character Assets
https://github.com/eveseat/api/blob/master/src/Http/Controllers/Api/v2/CharacterController.php#L210-L211

Reply:
Will do :)